packages/react/src/components/Card/Card.test.jsx

```diff
@@ -417,6 +417,31 @@ describe('Card', () => {
       expect(tooltipButton).toHaveAttribute('aria-expanded', 'true');
     });
 
+    it('should remove the tooltip if the title changes to a shorter string', async () => {
+      const aLongTitle =
+        'A very very long title which will almost certainly overflow and require a tooltip and we must test these things, you know.';
+
+      const aShortTitle = 'A Title';
+      const { rerender } = render(<Card {...cardProps} title={aLongTitle} />);
+      const tooltipButton = screen.getByRole('button', {
+        name: aLongTitle,
+      });
+      expect(tooltipButton).toBeVisible();
+      expect(tooltipButton).toHaveClass(`${iotPrefix}--card--title--text__overflow`);
+      Object.defineProperty(HTMLElement.prototype, 'clientWidth', {
```

Comment:
Do we need to reset these prototypes to prevent them from possibly spilling over on other tests?

Reply:
We do reset them in an `afterAll` on line 401

Reply:
I don't see `clientWidth` being reset there